apps/console/src/features/organizations/components/add-organization-modal.tsx

```diff
@@ -0,0 +1,298 @@
+/**
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AlertLevels, TestableComponentInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import { Field, Form } from "@wso2is/form";
+import { Heading, LinkButton, PrimaryButton } from "@wso2is/react-components";
+import React, { FunctionComponent, ReactElement, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Divider, Grid, Message, Modal, Form as SemanticForm } from "semantic-ui-react";
+import { EventPublisher } from "../../core";
+import { addOrganization, getOrganizations } from "../api";
+import { ORGANIZATION_TYPE } from "../constants";
+import { AddOrganizationInterface, OrganizationInterface, OrganizationListInterface } from "../models";
+
+interface OrganizationAddFormProps {
+    name: string;
+    domainName?: string;
+    description?: string;
+}
+
+/**
+ * Prop types of the `AddOrganizationModal` component.
+ */
+interface AddOrganizationModalPropsInterface extends TestableComponentInterface {
```

Comment:
Lets use `IdentifiableComponentInterface` instead of the depreacated one.
```suggestion
interface AddOrganizationModalPropsInterface extends IdentifiableComponentInterface {
```